src/model/transactions.coffee

```diff
@@ -59,3 +61,14 @@ TransactionSchema.index "request.timestamp"
 
 #compile schema into Model
 exports.Transaction = connectionDefault.model 'Transaction', TransactionSchema
+
+exports.Transaction.collection.createIndex({
+  'request.timestamp': 1
+  channelID: 1
+},
+{
+  name: 'transaction.autoRetry'
```

Comment:
Why does this need to be a partial index? Should we not just permanently add these indexes for all transactions? Both seems useful?